wye/base/management/commands/sample_data.py

```diff
@@ -20,37 +24,53 @@
 
 
 class Command(BaseCommand):
+    help = "Creating Initial demo data for testing application"
     fake = Faker()
 
     @transaction.atomic
     def handle(self, *args, **options):
         self.fake.seed(4321)
 
-        print('  Updating domain to localhost:8000')  # Update site url
+        self.stdout.write('  Updating domain to localhost:8000')  # Update site url
```

Comment:
Note

When you are using management commands and wish to provide console output, you should write to self.stdout and self.stderr, instead of printing to stdout and stderr directly. By using these proxies, it becomes much easier to test your custom command. Note also that you don’t need to end messages with a newline character, it will be added automatically, unless you specify the ending parameter:

self.stdout.write("Unterminated line", ending='')

(Source)[https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/]